lib/RollingFileWriteStream.js

```diff
@@ -246,7 +248,7 @@ class RollingFileWriteStream extends Writable {
     debug("_clean: existing files are: ", existingFileDetails);
     if (this._tooManyFiles(existingFileDetails.length)) {
       const fileNamesToRemove = existingFileDetails
-        .slice(0, existingFileDetails.length - this.options.numToKeep - 1)
+        .slice(0, existingFileDetails.length - this.options.numToKeep)
```

Comment:
This doesn't seem to be impacted by the exposure of `fileNameSep`.
Any reason/issue that requires this to be changed?

Reply:
This portion seems to alter the current behavior.

### From:
If `numBackups=3` then there will be 4 files:
```
filename
filename.1
filename.2
filename.3
```

### To:
If `numBackups=3` then there will be 3 files:
```
filename
filename.1
filename.2
```

With this, it seems to allow for 0 backups which is mentioned in #59.

Reply:
Reverted the logic change.